s29862-JuliaButkiewicz/Zadania_4/RozwiązaniaZadań4.txt

```diff
@@ -51,4 +52,60 @@ CTRL + \ <-- co powinno skutecznie zabić proces. Lub mogłabym
 użyć komendy kill, gdybym znała tylko id danego procesu.
 
 Zadanie 5.
+Jak wypisać wszystkie pliki w bieżącym katalogu, których nazwa zaczyna
+się od "al"?
+
+ROZWIĄZANIE:
+Szukając rozwiązania zastąpiłam 'al' --> 'in', gdyż wtedy
+faktycznie wyszukiwało mi pliki. Wpisałam więc, bazując
+na wysłanych materiałach komendę: ls -p | grep '^in[^/]*$'.
```

Comment:
Bardzo ładnie. Jednakże by było to 100% poprawne to dobrze jest do grepa dodać parametr `-e` włączający obsługę wyrażeń regularnych. Zdaje się, że od pewnego czasu jest ona jakoś dodawana automatycznie acz dobrze jest pamiętać o tym, że nie zawsze będzie się pracowało na nowych systemach i wtedy podanie explicite parametru `-e` jest wymagane. Zatem lepiej jest utrwalać sobie istnienie i potrzebę podawania tego parametru.